src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocblocktaglocation/InputJavadocBlockTagLocationIncorrect.java

```diff
@@ -0,0 +1,17 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocblocktaglocation;
+
+/**
+ * configuration: default
+ */
+public class InputJavadocBlockTagLocationIncorrect {
+
+    /**
+     * Summary. @author me
+     * <p>Text</p> @since 1.0
+     * @param param a parameter @param, @custom
+     * text @return one more @throws text again.
```

Comment:
can we use some key-word to show user where violation will be placed.
Whole idea of one config for one test is to place violations words in file, so it is possible to validate behavio simply looking at input file.

Reply:
done